tutorials/tutorial1.ipynb

```diff
@@ -33,12 +34,12 @@
    "metadata": {},
    "source": [
     "## Step 1: Create folder and set AtacWorks path\n",
-    "Replace 'path_to_atacworks' with the path to your cloned and set up 'AtacWorks' github repository."
+    "Replace 'atacworks' with the path to your cloned and set up 'AtacWorks' github repository."
```

Comment:
It's not `atacworks` that should be replaced but `/ntadimeti/AtacWorks`.

Reply:
changed it to "path_to_atacworks"